Basic Styles and App finished.

victorjv37 opened the pull request:
New styles for EpisodeData.
PodcastData bug fixed.
Fixed styles in PodcastData.
Fixed podcastList.js

victorjv37 committed:
Basic Styles and App finished.

New styles for EpisodeData.
PodcastData bug fixed.
Fixed styles in PodcastData.
Fixed podcastList.js

victorjv37 committed:
Better styling

New styles and organization for podcastData.

carloslozanosotoplexus commented on src/components/AudioPlayer.jsx:
```diff
@@ -0,0 +1,12 @@
+const AudioPlayer = ({ audioSrc }) => {
+  return (
+    <div>
+      <audio controls>
```
Pending more information and structured card

carloslozanosotoplexus commented on src/components/EpisodeData.jsx:
```diff
@@ -1,13 +1,13 @@
+import AudioPlayer from "./AudioPlayer";
+
 const EpisodeData = ({ podcastFound }) => {
```
pass through only episode information rather than array to set only index 0

victorjv37 committed:
Applied checking changes

Added styles for EpisodeView.
Added views folder and components.
Changed podcastEpisode variable on EpisodeData component.
Deleted div on Audioplayer component.
